### PR TITLE
Add Static Range

### LINKED ISSUE #19: Add static Range method
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.range?view=netcore-2.1

### PR DESCRIPTION
**Pull request by PieceMaker**
Fixes #19.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ interface Array<T> {
 
 interface ArrayConstructor {
   Empty(): Array<any>;
+  Range(): Array<number>;
```
Comment (jreina) [inline index.d.ts]: This is missing the `start` and `count` parameters.

File: src/Range.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  if(!Number.isInteger(start))
+  {
+    throw '"start" must be an integer';
+  }
+  if(!Number.isInteger(count))
+  {
+    throw '"count" must be an integer';
+  }
+  const emptyArray = new Array(count);
+  return emptyArray.fill(start).map(_add);
```
Comment (jreina) [inline src/Range.js:31]: This is pretty clever.

**Review by jreina — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (PieceMaker) [reply index.d.ts]: Try now.

**Review by jreina — APPROVED** [review]

**Comment by jreina** [comment]
Thank you for your contribution. Happy hacking 🎃

**Pull request merged**